resources/views/MyAccount/Components/OrderHistory.twig

```diff
@@ -95,7 +95,8 @@
                                                 {{ trans("Ceres::Template.orderHistoryOrderDetails") }}
                                             </button>
                                             {% set returnOrder = entry.toArray() %}
-                                            {% if returnOrder.isReturnable %}
+                                            {% set allowReturns = config("Ceres.my_account.order_return_active") == "true" %}
+                                            {% if returnOrder.isReturnable and allowReturns %}
```

Comment:
```suggestion
                                            {% if returnOrder.isReturnable and ceresConfig.myAccount.orderReturnActive %}
```

Die `PluginConfig`-Klasse regelt bei boolean-Flags mehrere Fälle, da je nach Konstellation `"true"` oder `"1"` oder `1` in der Tabelle gespeichert werden.